source/playgrounds.txt

```diff
@@ -121,6 +122,25 @@ panel, the |vsce| *Overview* page, or the |vscode| Command Pallette.
 
       .. include:: /includes/steps/open-new-playground.rst
 
+   .. tab:: Your Project File
+      :tabid: project-file
+
+      1. Open a project file.
```

Comment:
Needs to be more specific, something like:
"Open your application project file that has the MongoDB query you want to export"